android/build.gradle

```diff
@@ -23,6 +23,21 @@ allprojects {
         maven { url "$rootDir/../node_modules/react-native/android" }
     }
 
+    // Make sure we use the react-native version in node_modules and not the one
```

Comment:
Can you give more details about this issue ? Is it that some dependency projects use other version that we declare or what is it about exactly ?

Reply:
I mentioned it in the description, but I'll rephrase: jcenter has react native 0.55 published, and our dependencies don't have our maven repo as the first option, so that version is chosen.

This way we are forcing the entire dependency chain to use the RN version we have decided with package.json.